x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/parser/LogicalPlanBuilder.java

```diff
@@ -11,20 +11,16 @@
 import org.elasticsearch.xpack.ql.expression.UnresolvedAttribute;
 import org.elasticsearch.xpack.ql.expression.predicate.logical.And;
 import org.elasticsearch.xpack.ql.expression.predicate.operator.comparison.Equals;
-import org.elasticsearch.xpack.ql.index.EsIndex;
-import org.elasticsearch.xpack.ql.plan.logical.EsRelation;
 import org.elasticsearch.xpack.ql.plan.logical.Filter;
 import org.elasticsearch.xpack.ql.plan.logical.LogicalPlan;
+import org.elasticsearch.xpack.ql.plan.logical.UnresolvedRelation;
 import org.elasticsearch.xpack.ql.tree.Source;
 import org.elasticsearch.xpack.ql.type.DataTypes;
 
-import static java.util.Collections.emptyMap;
-
 public abstract class LogicalPlanBuilder extends ExpressionBuilder {
 
     // TODO: these need to be made configurable
-    private static final String EVENT_TYPE = "event.category";
-    private static final EsIndex esIndex = new EsIndex("<not-specified>", emptyMap());
+    private static final String EVENT_TYPE = "event_type";
```

Comment:
I changed this since the data sample from existing EQL documentation uses `event_type`. If the default field should be changed, that is best addressed separately:
- figure out what's the impact on backwards compatibility (ECS schema) and existing data
- update datasets
- update code (last).

Reply:
was assuming that the data will be in ECS format
```{
    "user": {
        "group": {}
    },
    "host": {
        "os": {
            "platform": "windows",
            "name": "Windows"
        },
        "ip": "127.0.0.1",
        "hostname": "localhost",
        "name": "localhost"
    },
    "event": {
        "module": "endgame",
        "dataset": "esensor",
        "action": "already_running",
        "category": "process",
        "kind": "event"
    }, 
```
thus the event.category for EVENT_TYPE